### DISCUSSION TITLE
Installazione HA su PC

### DISCUSSION THREAD

**Discussion by magomer** [indomus/forum#271] [HUB: Home Assistant]
Buongiorno a tutti.
E' qualche settimana che sto cercando di avvicinarmi ad HA. Ho provato le varie installazioni suggerite su PC, sia come VM sotto Windows 10, sia come docker con Debian, proxmox etc etc
Il problema è SUPERVISOR e la propria repository dei vari ADDON ufficiali.
Anche quando parto con il sistema OK, quindi supervisor completo e funzionante, dopo le vari installazioni di Duck, file editor, SSH e più in generale degli addon basici, Supervisor si "svuota", non mi fa più vedere le integrazioni possibili. Quello che ho potuto verificare è che mentre prima nelle repositori mi riconosceva il nome dell'autore, adesso mi da UNKNOW e nel registro di SU viene fuori una scritta tipo : "supervisor.store.repository] can't remove built-in repositories!" più o meno.... Prima di abbandonare TUTTO, avete idee e/o suggerimenti ? 
Ripeto che sto parlando di installazione su PC, avendo un portatile "decente" a completa disposizione.
Grazie !

**Comment by Jones1982** [comment]
Ciao, avevo avuto un problema simile poco tempo fa...
prova a dare un occhiata qui al mio post:

[https://spectrum.chat/indomus/home-assistant/aggiornamento-core-da-2021-2-3-a-2021-3-0-e-2021-3-1-errori-risolto~103c3666-7084-4128-b243-0beae3ea57ea](https://spectrum.chat/indomus/home-assistant/aggiornamento-core-da-2021-2-3-a-2021-3-0-e-2021-3-1-errori-risolto~103c3666-7084-4128-b243-0beae3ea57ea)
